inc/cxplat.hpp

```diff
@@ -0,0 +1,87 @@
+/*++
+
+    Copyright (c) Microsoft Corporation.
+    Licensed under the MIT License.
+
+Abstract:
+
+    C++ header only cxplat wrappers
+
+--*/
```

Comment:
Add a `#pragma once` too, before `#ifndef CXPLATCPP_H`

Reply:
why is this necessary though? `#ifndef CXPLATCPP_H` already guarantees that the header is used once. Does `pragma` make compile faster or something?

Reply:
pragmas are faster AFAIU